types/custom-functions-runtime/index.d.ts

```diff
@@ -86,7 +98,11 @@ declare namespace CustomFunctions {
     interface CancelableInvocation extends Invocation {
         /**
          * Event handler called when the custom function is canceled.
+         *
+         * @remarks
          * [Api set: CustomFunctionsRuntime 1.1]
+         *
+         * @eventproperty
```

Comment:
@ElizabethSamuel-MSFT Can you confirm that @eventproperty is the right tag to display this in the Events section?

Reply:
@alison-mk Yes, @eventproperty is the correct tag